docs/photon/deployment.md

```diff
@@ -2,10 +2,40 @@
 
 Photon.js depends on a query engine that's running as a binary on the same host as your application. When deploying your Photon-based application to production, you need to ensure that the binary used by Photon can run in your production environment, i.e. it needs to be compatible with the runtime of your deployment provider.
 
-The query engine binary is downloaded when you run `prisma2 generate`, it is then stored alongside the generated Photon code inside `node_modules/@generated` (or the [custom `output` path](./codegen-and-node-setup.md) you specified). 
+The query engine binary is downloaded when you run `prisma2 generate`, it is then stored alongside the generated Photon code inside `node_modules/@generated` (or the [custom `output` path](./codegen-and-node-setup.md) you specified).
 
 **IMPORTANT**: To ensure the query engine binary is compatible with your production environment, you have to [specify the right platform for Photon.js](../core/generators/photonjs.md#specifying-the-right-platform-for-photon-js).
 
+## Photon in FaaS environment (Like AWS Lambda)
+
+**DB Connection Handling**
```

Comment:
use `###` headings instead of bold text for subheadings

Reply:
also, please use regular casing instead of title casing for subheadings and spell out `database`: `Database connection handling`

Reply:
```suggestion
### Database connection handling
```

Reply:
@nikolasburk I did this in md file but like Jan is doing it, you can also suggest changes that I can actually commit :)